### PR TITLE
Updated the About Page & added Our Team Page

### PR DESCRIPTION
**Pull request by au-shah**
We will need to change the picture comparing the PM2.5/10 particles once @JonahIBM has created his own

### CODE REVIEW

File: ClimateMonitorV2/views/about.pug [hunk 1]
```diff
@@ -9,39 +10,69 @@ block content
...
+                        tr
+                          td Organic Compounds
+                          td Pollen
+                        tr
+                          td Metals
+                          td Mould
+
+                div(class='more-info')
+                    h3() Size of these pollutants put into perspective:
+                    img(class="img-fuild human-comp", src="https://ww2.arb.ca.gov/sites/default/files/inline-images/human_hair_0.png")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/about.pug]: Is this supposed to be fuild?

File: ClimateMonitorV2/views/our-team.pug [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+              li.col-12.col-md-6.col-lg-3
+                .cnt-block.equal-hight(style='height: 349px;')
+                  figure
+                    img.img-responsive(src='images/akshat.png' alt='')
+                  h3
+                    a Akshat Shah
+                  p Web Designer &amp; Documentation
+                  ul.follow-us.clearfix
+                    li
+                      a(href='https://github.com/au-shah')
```
Comment (M-Davies) [inline ClimateMonitorV2/views/our-team.pug]: Can we add `target="_blank"` to each of our github and linkedin `a` tags? That way, they open in a new tab and it's a whole lot more nice doing that than having to click the back button several times 😉

File: ClimateMonitorV2/views/index.pug [hunk 3]
```diff
@@ -21,14 +21,17 @@ block content
...
                 a(class="nav-link" href="/about") About
             div(class="sidebar-section")
-                img(class="sidebar-icon" src="https://img.icons8.com/ios/20/000000/statistics.png")
+                img(class="sidebar-icon" src="icons/statistics.svg")
                 a(class="nav-link" href="/detailedstats") Statistics
+            div(class="sidebar-section")
+                img(class="sidebar-icon" src="icons/group.svg")
+                a(class="nav-link" href="/our-team") Our Team
             div(class="sidebar-footer")
                 a(href="https://urbanflows.ac.uk/" target="_blank")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/index.pug:36]: Urban flows logo is really far down in the sidebar footer on all pages again, like it's falling off the page. Could we add some css to push it up a bit?

**Review by M-Davies — CHANGES_REQUESTED** [review]
This looks awesome! I love team page and the new icons 😍. Just a couple of nitpicks (it wouldn't be me if I didn't do it) and it'll look perfect

**Comment by M-Davies** [comment]
Chrome also wants to translate the page into Norwegian which is kind of funny.

**Comment by au-shah** [comment]
> Chrome also wants to translate the page into Norwegian which is kind of funny.

I think I saw this once too - not too sure what is causing this 🤷

**Comment by M-Davies** [comment]
Eh. I don't think it matters too much. Maybe there will be a Norwegian geography student at Sheffield who wants to know how awful the air is on a certain day

**Review by M-Davies — APPROVED** [review]
Awesome sauce. I approve

**Pull request merged**